_opensearch/install/compatibility.md

```diff
@@ -17,3 +17,5 @@ nav_order: 2
 OpenSearch version | Compatible Java versions | Recommended operating systems
 :--- | :--- | :---
 1.x | 8, 11, 15 | Red Hat Enterprise Linux 7, 8; CentOS 7, 8; Amazon Linux 2; Ubuntu 16.04, 18.04, 20.04
+
+If you plan on running several applications each with different Java requirements, set the `OPENSEARCH_JAVA_HOME` to the location of your Java install. The `OPENSEARCH_JAVA_HOME` supersedes the `JAVA_HOME` environment variable.
```

Comment:
```suggestion
If you plan on running several applications each with different Java requirements, set the `OPENSEARCH_JAVA_HOME` to the location of your Java install. 
```
@Naarcha-AWS it does not correctly reflect the intent: there is more than one Java installation, the user could be pick any for Opensearch.

```suggestion
If you plan on running several applications each with different Java requirements, set the `OPENSEARCH_JAVA_HOME` to the location of the Java installation to be used by Opensearch.
```
WDYT?

Reply:
I see. What do you think of, "If you plan on running applications besides OpenSearch with different Java requirements, set the `OPENSEARCH_JAVA_HOME` environment variable to the location of the Java installation to be used by OpenSearch."

Reply:
Done, thanks @Naarcha-AWS !